### PR TITLE
created a airbnb experiences clone.

### PR DESCRIPTION
**Pull request by snehapoulose**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import Card from "./components/Card";
+import data from "./data.json";
+import "./App.css";
+import Navbar from "./components/Navbar";
+
+function App() {
+  const cards = data.map((item) => {
+    return <Card key={item.id} {...item} />;
+  });
+  return (
```
Comment (dheeraj-murali) [inline src/App.js:12]: adding a line break here would be nice, so that we know these are 2 separate blocks of code

File: src/components/Card.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "../App.css";
+function Card(props) {
```
Comment (dheeraj-murali) [inline src/components/Card.js:4]: line break

**Review by dheeraj-murali — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (snehapoulose) [reply src/App.js:12]: added line break.

File: src/components/Card.js (hunk 2 above)
Comment (snehapoulose) [reply src/components/Card.js:4]: added line break.

**Pull request merged**